CHANGELOG.md

```diff
@@ -37,6 +37,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [v0.47.7](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.47.7) - 2023-12-18
```

Comment:
Still curious if we should include https://github.com/cosmos/cosmos-sdk/pull/18714.